config/config.go

```diff
@@ -69,6 +73,12 @@ func NewConfig(requiredVars []string) Config {
 		errString = fmt.Sprintf("missing required variables: %v", missingVars)
 	}
 
+	if output == "" {
+		output = "yaml"
+	} else if _, ok := allowedOutputTypes[output]; !ok {
```

Comment:
any interest in doing `slice.Contains` here instead, with a basic `[]string` for allowedOutputTypes??